src/main/java/core/basesyntax/service/operation/BalanceOperationHandler.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.StorageService;
+
+public class BalanceOperationHandler implements OperationHandler {
+    private final StorageService storageService;
+
+    public BalanceOperationHandler(StorageService storageService) {
+        this.storageService = storageService;
+    }
+
+    @Override
+    public void handle(FruitTransaction transaction, StorageService storageService) {
+        storageService.addFruit(transaction.getFruit(),transaction.getQuantity());
+    }
```

Comment:
This class should not have a dependency on `StorageService`. According to the checklist, all services should be independent and called in the `main()` method. The `handle` method should not take `StorageService` as a parameter, but instead work with the `Storage` directly.